src/Generation/GapicClientGenerator.php

```diff
@@ -825,79 +783,32 @@ private function rpcMethod(MethodDetails $method): PhpClassMember
-            $optionalArgs->var,
-            $request,
-            AST::call(AST::THIS, AST::method('getOperationsClient'))(),
-            AST::NULL,
-            AST::access($this->ctx->type($method->responseType), AST::CLS),
-        ];
-        return AST::call(AST::THIS, AST::method('startOperationsCall'))(...$startCallArgs)->wait();
-      case MethodDetails::NORMAL:
-        $startCallArgs[] = $request;
-        if ($method->isMixin()) {
-            $startCallArgs[] =
-                AST::access($this->ctx->type(Type::fromName(Call::class)), AST::constant('UNARY_CALL'));
-            $startCallArgs[] = $method->mixinServiceFullname;
-        }
-        return AST::call(AST::THIS, AST::method('startCall'))(...$startCallArgs)->wait();
-      case MethodDetails::LRO:
-        $startCallArgs = [
-          $method->name,
-          $optionalArgs->var,
-          $request,
-          AST::call(AST::THIS, AST::method('getOperationsClient'))()
-        ];
-        if ($method->isMixin()) {
-            $startCallArgs[] = $method->mixinServiceFullname;
-        }
-        return AST::call(AST::THIS, AST::method('startOperationsCall'))(...$startCallArgs)->wait();
-      case MethodDetails::PAGINATED:
-        $startCallArgs = [
-          $method->name,
-          $optionalArgs->var,
-          AST::access($this->ctx->type($method->responseType), AST::CLS),
-          $request
-        ];
-        if ($method->isMixin()) {
-            $startCallArgs[] = $method->mixinServiceFullname;
-        }
-        return AST::call(AST::THIS, AST::method('getPagedListResponse'))(...$startCallArgs);
       case MethodDetails::BIDI_STREAMING:
-        $startCallArgs[] = AST::NULL;
-        $startCallArgs[] =
-          AST::access($this->ctx->type(Type::fromName(Call::class)), AST::constant('BIDI_STREAMING_CALL'));
-        if ($method->isMixin()) {
-            $startCallArgs[] = $method->mixinServiceFullname;
-        }
-        return AST::call(AST::THIS, AST::method('startCall'))(...$startCallArgs);
-      case MethodDetails::SERVER_STREAMING:
-        $startCallArgs[] = $request;
-        $startCallArgs[] =
-          AST::access($this->ctx->type(Type::fromName(Call::class)), AST::constant('SERVER_STREAMING_CALL'));
-        if ($method->isMixin()) {
-            $startCallArgs[] = $method->mixinServiceFullname;
-        }
-        return AST::call(AST::THIS, AST::method('startCall'))(...$startCallArgs);
+        // Fall through to CLIENT_STREAMING.
       case MethodDetails::CLIENT_STREAMING:
-        $startCallArgs[] = AST::NULL;
-        $startCallArgs[] =
-          AST::access($this->ctx->type(Type::fromName(Call::class)), AST::constant('CLIENT_STREAMING_CALL'));
-        if ($method->isMixin()) {
-            $startCallArgs[] = $method->mixinServiceFullname;
-        }
-        return AST::call(AST::THIS, AST::method('startCall'))(...$startCallArgs);
+        $startApiCallArgs = $startApiCallArgs->set('request', AST::NULL);
```

Comment:
I realize the indentation was off prior to this change, but would you be open to cleaning it up here?

Reply:
Ha I don't mind at all, I was wondering why it was like that assuming the formatter did it. Done.